src/Symfony/Component/Debug/Tests/Fixtures/FinalMethod.php

```diff
@@ -13,10 +13,8 @@ public function finalMethod()
 
     /**
      * @final
-     *
-     * @return int
      */
-    public function finalMethod2()
+    public function finalMethod2(): int
```

Comment:
I would recommend to revert changes to fixture files, because some of the might actually need the return to be missing to correctly test what they should test, and I don't know which ones do :)
+ it's OK not adding return types to them of course.